lib/src/internal/_HttpEndpoints.dart

```diff
@@ -1865,4 +1865,4 @@ class _HttpEndpoints implements IHttpEndpoints {
   @override
   String getRoleIconUrl(Snowflake roleId, String iconHash, String format, int size) =>
       "${Constants.cdnUrl}/role-icons/$roleId/$iconHash.$format?size=$size";
-}
+}
```

Comment:
Removing trailing new line wasn't needed tho. That should stay in place

Reply:
https://github.com/nyxx-discord/nyxx/blob/1ea7a859168c6fcabfcfa4778fb1dd3d1f97c7cc/lib/src/internal/_HttpEndpoints.dart#L1865-L1868

There wasn't a trailing new line in the first place

Reply:
there is but github doesnt show that, but its should lack of new line. Besides if new line wansnt present to begin with diff woudn't show that line as changed
![image](https://user-images.githubusercontent.com/23033957/141790461-f6677836-5eb8-4219-81c3-83018945675e.png)

Reply:
Didn't know that. Added trailing new line

Reply:
thanks